python_ga/genetic_algorithm.py

```diff
@@ -872,40 +168,40 @@ def new_genetic_algorithm(population, model, config, converter):
     logger.info('Initialize computation')
     
     date1 = datetime.now()
-    pop_fitness = calculate_fitness(population=population, model=model, converter=converter)
+    paired_population = converter.convert_population_lists_to_pairs(protocols=population)
+    pop_fitness = calculate_fitness(paired_population=paired_population, model=model)
```

Comment:
`calculate_fitness` now works directly on `paired_population`, without internal conversion in `calculate_fitness`